<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign</string>
  </property>
  <widget class="QComboBox" name="CMB_userId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="User">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_roleId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_userRoleStatus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_assignRole">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Assign Role</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_userName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>user name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_roleName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>role name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
